test/unit/src/common/reducers/t_snap-builds.js

```diff
@@ -72,93 +73,84 @@ describe('snapBuilds reducers', () => {
   });
 
   context('FETCH_BUILDS', () => {
-    it('should store fetching status when fetching builds', () => {
-      const action = {
-        type: ActionTypes.FETCH_BUILDS,
-        payload: 'test'
-      };
+    const action = {
+      type: ActionTypes.FETCH_BUILDS,
+      payload: 'test'
+    };
 
+    it('should store fetching status when fetching builds', () => {
       expect(snapBuilds(initialState, action)).toEqual({
         ...initialState,
         isFetching: true
       });
     });
   });
 
-  context('FETCH_BUILDS_SUCCESS', () => {
+  context('FETCH_SNAP_SUCCESS', () => {
+    const SNAP_LINK = 'https://api.launchpad.net/devel/~cjwatson/+snap/godd-test-2';
+
+    const state = {
+      ...initialState,
+      isFetching: true
+    };
+    const action = {
+      type: ActionTypes.FETCH_SNAP_SUCCESS,
+      payload: SNAP_LINK
+    };
 
     it('should stop fetching', () => {
-      const state = {
-        ...initialState,
-        isFetching: true
-      };
+      expect(snapBuilds(state, action).isFetching).toBe(false);
+    });
 
-      const action = {
-        type: ActionTypes.FETCH_BUILDS_SUCCESS,
-        payload: SNAP_ENTRIES
-      };
+    it('should store snap link', () => {
+      expect(snapBuilds(state, action).snapLink).toEqual(SNAP_LINK);
+    });
+  });
+
+  context('FETCH_BUILDS_SUCCESS', () => {
+    const state = {
+      ...initialState,
+      isFetching: true,
+      error: 'Previous error'
```

Comment:
Added `error` to also cover "it should clean error" test case as mentioned in review comments.